src/main/webapp/resources/js/validation-date.js

```diff
@@ -0,0 +1,28 @@
+function validateBookingsDate() {
+
+    var fromDateValue = $('#from').val();
+    var toDateValue = $('#to').val();
+    var regex = /^(\d{4})(\/|-)(\d{1,2})(\/|-)(\d{1,2})$/;
+    var isValidate = true;
+
+    if (fromDateValue.match(regex) == null) {
+        nowYear = new Date().getFullYear();
+        month = fromDateValue.split("-")[1];
+        day = fromDateValue.split("-")[2];
+        $('#from').val(nowYear + "-" + month + "-" + day);
+        isValidate = false;
+    } else if (toDateValue.match(regex) == null) {
+        nowYear = new Date().getFullYear();
+        month = toDateValue.split("-")[1];
+        day = toDateValue.split("-")[2];
+        $('#to').val(nowYear + "-" + month + "-" + day);
```

Comment:
it is "input type="date""
we should use regex because user can write more than 4 digits in year field.
user could write only incorrect year in this form, that is why i set year to current year when the year is invalid.
I think that it is always possible to split because of standard value format of html tag "input date"